src/test/scala/me/rexim/morganey/AutocompletionSpec.scala

```diff
@@ -16,7 +16,7 @@ class AutocompletionSpec extends FlatSpec with Matchers with TestTerms  {
   def autocomplete(line: String, cursor: Int, knownNames: List[String]): Set[CharSequence] = {
     val id = I(lvar("x"))
     val fakeBindings = knownNames.map(name => MorganeyBinding(lvar(name), id))
-    val moduleFinder = new ModuleFinder(List(new File("./std/")))
+    val moduleFinder = new ModuleFinder(List(new File("./test-data/load-autocomplete/")))
```

Comment:
Very nice idea to use a different folder for the autocompletion-tests.

Aren't test resources in a project with maven directory structure located at `src/test/resources`?